website/docs/concepts/html/elements.mdx

```diff
@@ -118,86 +118,6 @@ html! {
 
 If the attribute is set to `None`, the attribute won't be set in the DOM.
 
-## Listeners
-
-Listener attributes need to be passed a `Callback` which is a wrapper around a closure. How you create your callback depends on how you wish your app to react to a listener event:
-
-<Tabs>
-  <TabItem value="Component handler" label="Component handler">
-
-```rust
-use yew::{Component, Context, html, Html};
-
-struct MyComponent;
-
-enum Msg {
-    Click,
-}
-
-impl Component for MyComponent {
-    type Message = Msg;
-    type Properties = ();
-
-    fn create(_ctx: &Context<Self>) -> Self {
-        Self
-    }
-
-    fn update(&mut self, _ctx: &Context<Self>, msg: Self::Message) -> bool {
-        match msg {
-            Msg::Click => {
-                // Handle Click
-            }
-        };
-        true
-    }
-
-    fn view(&self, ctx: &Context<Self>) -> Html {
-        // Create a callback from a component link to handle it in a component
-        let click_callback = ctx.link().callback(|_| Msg::Click);
-        html! {
-            <button onclick={click_callback}>
-                { "Click me!" }
-            </button>
-        }
-    }
-}
-```
-
-  </TabItem>
-  <TabItem value="Other Cases" label="Other Cases">
-
-```rust
-use yew::{Callback, Context, Component, html, Html};
-use weblog::console_log;
-
-struct MyComponent;
-
-impl Component for MyComponent {
-    type Message = ();
-    type Properties = ();
-
-    fn create(_ctx: &Context<Self>) -> Self {
-        MyComponent
-    }
-
-    fn view(&self, _ctx: &Context<Self>) -> Html {
-        // Create an ephemeral callback
-        let click_callback = Callback::from(|_| {
-            console_log!("clicked!");
-        });
-
-        html! {
-            <button onclick={click_callback}>
-                { "Click me!" }
-            </button>
-        }
-    }
-}
-```
-
-  </TabItem>
-</Tabs>
-
```

Comment:
Is this section not needed anymore?

Reply:
We handle this in callbacks doc, so yes no longer needed